pkg/types/config/parser.go

```diff
@@ -33,7 +33,7 @@ func ParseConfig(data []byte) (*Cluster, error) {
 		cluster.PullSecret = string(data)
 	}
 
-	if cluster.EC2AMIOverride == "" {
+	if cluster.Platform == "aws" && cluster.EC2AMIOverride == "" {
```

Comment:
Can you use [`PlatformAWS`][1]?

[1]: https://github.com/openshift/installer/blob/29e8376dfe8acf49e823e21b3d9fefbc5c459626/pkg/types/config/cluster.go#L21

Reply:
done!